### PR TITLE
Permalink by Query Param instead of Hash

### PR DESCRIPTION
**Pull request by okbel**
## What does this PR do?
- Uses `commentId` query param instead of hash
- Solves error while using wrong commentId

## How do I test this PR?

- Go to the Stream 
- Select a comment
- Check the permalink (you should have `talk-plugin-permalink` enabled)
- Open that in another tab
- Replace that commentId by a fake one

### CODE REVIEW

File: client/coral-embed-stream/src/components/StreamError.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import styles from './StreamError.css';
+
+export const StreamError = () => (
+  <div className={styles.streamError}>
+    An error has occured!
```
Comment (alinhle) [inline client/coral-embed-stream/src/components/StreamError.js]: Translation?

**Comment by kgardnr** [comment]
This works great!

Question -- @okbel @cvle @jde -- now that we're using the query param, do we need to update where we reference permalinks elsewhere on Talk, e.g. on My Profile tab, and View Context link from admin?

File: client/coral-embed/src/index.js [hunk 2]
```diff
@@ -190,7 +190,10 @@ Talk.render = function(el, opts) {
   // Compose the query to send down to the Talk API so it knows what to load.
   let query = {};
 
-  query.comment_id = window.location.hash.slice(1);
+  let urlParams = new URLSearchParams(window.location.search);
```
Comment (alinhle) [inline client/coral-embed/src/index.js:194]: Support for `URLSearchParams` is not good, see: http://caniuse.com/#feat=urlsearchparams.
Needs alternative or polyfill.

Comment (okbel) [reply client/coral-embed/src/index.js:194]: I checked that: Are we supporting IE or Opera Mini? /cc @kgardnr

Comment (kgardnr) [reply client/coral-embed/src/index.js:194]: Ah, we do need to support IE unfortunately @okbel

Comment (okbel) [reply client/coral-embed/src/index.js:194]: Adding that polyfill right now!

File: client/coral-embed-stream/src/components/StreamError.js (hunk 1 above)
Comment (okbel) [reply client/coral-embed-stream/src/components/StreamError.js]: Yes! Adding that right now!

**Comment by alinhle** [comment]
@kgardnr @okbel I think so. The `view context` links must probably be adapted.

**Comment by okbel** [comment]
@kgardnr @cvle Ready to be tested!

**Comment by kgardnr** [comment]
Just need to update "View Context" link on the comment cards in Admin and we're good to go!
talk_-_coral_admin

**Comment by okbel** [comment]
@kgardnr I added that one too! That one should be fixed too :)

**Review by alinhle — APPROVED** [review]
🎉

**Pull request merged**